### ISSUE TITLE
can i assign a training and test data

### ISSUE THREAD

**Issue by Chigstardan** [martineastwood/penaltyblog#9]
so i used about 5 seasons worth of data that i got from footystats. i wanted to try to specify training data for the model and then use a test dataset so i can see the probabilities for each match on the test set instead of manually calling the predict function every time i want to get probabilities for a new team.

**Comment by martineastwood** [comment]
That's a good point - perhaps predict should optionally be able to take two iterables of team names and handle the iteration for the user. I'll add it to the TODO list

**Comment by martineastwood** [comment]
This will added in v0.8.0, which should be released in about a week
